### ISSUE TITLE
Touch Events

### ISSUE THREAD

**Issue by mstang** [quil/quil#226]
Hi,
I have been drawing on a canvas (clojurescript) and using a mouse to detect events.
How do I use Touch Events on Phones?

thanks,

Mark

**Comment by nbeloglazov** [comment]
Hi Mark. mouse-clicked, mouse-pressed, mouse-moved should work both on PC and phones. Here is example sketch that prints all handler events: http://quil.info/sketches/show/-L1oPMeKB76WmvqUZUTn When I open it on phone and click on sketch I see mouse-clicked events logged. 

Do you have a particular issue with mouse (touches) not working in quil?

**Comment by mstang** [comment]
Hi Nikita,
I looked through the ClojureScript code and didn't see anything referring to "touch".  And I couldn't figure out how to show a console on my phone.  And the defsketch only shows mouse functions.  So, I was very confused, which is why I posted the issue and the message.  I am getting an error message about "missing event handlers" in processing.js.  When I looked at the line that is generating the error I noticed it is in a function designed to handle the "touch" events.  I see there where it maps the touch events to mouse handlers.  So, I  updated one of my mouse handlers to draw some text when clicked.  And when I tap on my phone it appears.  So, I think everything is good.
Thanks for the quick reply.
